x/group/client/testutil/tx.go

```diff
@@ -2347,3 +2554,152 @@ func (s *IntegrationTestSuite) createCLIProposal(groupPolicyAddress, proposer, s
 
 	return testutil.WriteToNewTempFile(s.T(), string(bz)).Name()
 }
+
+func (s *IntegrationTestSuite) createAccounts(quantity int) []string {
```

Comment:
I think this function is useful. Can we make it so that other tests (maybe TestTxLeaveGroup or TestTxUpdateGroupMembers) also use this? For some DRY

Reply:
For sure